### ISSUE TITLE
incorrect/unused CSS property

### ISSUE THREAD

**Issue by GrayStrider** [react-boilerplate/react-boilerplate#2795]
## Description
https://github.com/react-boilerplate/react-boilerplate/blob/d19099afeff64ecfb09133c06c1cb18c0d40887e/app/containers/RepoListItem/Wrapper.js#L7
is incorrect:
![image](https://user-images.githubusercontent.com/43771776/67644274-3dcedb00-f974-11e9-8ac2-a4014a775e5d.png)
removing the property altogether doesn't seem to change anything, please double check.
Test snapshots update would be required.

## Steps to reproduce
IntelliJ WebStorm latest stable, code analysis

**Expected behavior**
No error, no unused properties

## Versions

- React-Boilerplate: 4.0
- Node/NPM: latest
- Browser: Chrome latest

**Comment by julienben** [comment]
Good catch! `space-between` is a valid value for `justify-content`, not `align-items`. This line can be safely removed.

Would you like to submit a PR to fix this? I'll take care of it otherwise.

**Comment by GrayStrider** [comment]
Will do!
